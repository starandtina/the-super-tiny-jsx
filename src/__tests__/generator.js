const tokenizer = require('../tokenizer')
const parser = require('../parser')
const generator = require('../generator')

it('parser', () => {
  expect(
    generator(
      parser(
        tokenizer(`
          html {
            body {
              h1 {
                "Hello World!"
              }
            }
          }
      `),
      ),
    ),
  ).toMatchSnapshot()
})
