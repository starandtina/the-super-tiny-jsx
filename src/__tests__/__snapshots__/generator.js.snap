// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`parser 1`] = `
"<!DOCTYPE html>
<html>
  <body>
    <h1>
      Hello World!
    </h1>
  </body>
</html>"
`;
